marklogic-data-hub-central/ui-custom/src/components/Membership/Membership.scss

```diff
@@ -0,0 +1,60 @@
+$background-color: #f4f4f4;
+$border-color: #ebebeb;
+$highlighted-color: #97e7d7;
+$title-color: #a2a2a2;
+
+.Membership {
+    display: flex;
+    flex-wrap: wrap;
+    .item {
+        flex-basis: 140px;
```

Comment:
Let's make the default of 12 lists fit. Let's also constrain the item container size so it doesn't get messy:
```
        flex-basis: 8.3%;
        min-width: 110px; 
        max-width: 160px; 
```